addons/binding/org.openhab.binding.fronius/src/main/java/org/openhab/binding/fronius/internal/FroniusHandlerFactory.java

```diff
@@ -8,20 +8,23 @@
  */
 package org.openhab.binding.fronius.internal;
 
-import static org.openhab.binding.fronius.FroniusBindingConstants.THING_TYPE_INVERTER;
+import static org.openhab.binding.fronius.FroniusBindingConstants.*;
 
-import java.util.Collections;
 import java.util.Set;
 
+import org.eclipse.smarthome.core.thing.Bridge;
 import org.eclipse.smarthome.core.thing.Thing;
 import org.eclipse.smarthome.core.thing.ThingTypeUID;
 import org.eclipse.smarthome.core.thing.binding.BaseThingHandlerFactory;
 import org.eclipse.smarthome.core.thing.binding.ThingHandler;
 import org.eclipse.smarthome.core.thing.binding.ThingHandlerFactory;
+import org.openhab.binding.fronius.handler.FroniusBridgeHandler;
 import org.openhab.binding.fronius.handler.FroniusHandler;
 import org.osgi.service.component.annotations.Component;
 import org.osgi.service.component.annotations.ConfigurationPolicy;
 
+import com.google.common.collect.ImmutableSet;
```

Comment:
Can you please remove the dependency to this library here, see my comment: https://github.com/openhab/openhab2-addons/pull/2344/files/97dd61dc3bcd2a9d558c0f8b2e032ecf80a497b5..6b21fac7339e7a8fe0e4b0864b68a3dbf9a5144c#r152042430